.travis.yml

```diff
@@ -21,11 +21,9 @@ jobs:
       after_script:
         - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
     - stage: Build & Publish
-      if: branch = master AND type != pull_request
+      if: branch IN ("master", "beta", "alpha") AND type != pull_request
       script: skip
       deploy:
         provider: script
         skip_cleanup: true
-        on:
-          branch: master
```

Comment:
Not sure why that would be mandatory here, since this step should only be run in the `master` context, that's why we removed it

Reply:
Maybe it was an unnecessary defensive condition.